pgcli/pgexecute.py

```diff
@@ -186,7 +185,9 @@ def connect(self, database=None, user=None, password=None, host=None,
 
             cursor = conn.cursor()
 
-        superuser = self._select_one(cursor, "select current_setting('is_superuser')::bool")[0]
+        cursor.execute("SHOW ALL")
```

Comment:
This seems such a broad stroke to check whether it is a super user or not. Are you sure there isn't a simpler way to do this?

Does `select current_setting('is_superuser')` not work in redshift? I thought the only issue in redshift is the typecasting of that value to bool.